skins/laika/src/components/pages/donation_form/Address.vue

```diff
@@ -5,7 +5,6 @@
 			<address-type></address-type>
 			<name :show-error="fieldErrors" :form-data="formData" :validate-input="validateInput" :address-type="addressType"></name>
 			<postal v-if="addressTypeIsNotAnon" :show-error="fieldErrors" :form-data="formData" :validate-input="validateInput" :countries="countries"></postal>
-			<h1 class="title is-size-1">{{ $t( 'donation_section_email_title' ) }}</h1>
```

Comment:
@tzhelyazkova Right now, email is a complete outlier here in this file / in the overall structure. If we do that for email, it should also be done for the other components, otherwise it does not really make sense.